mico-core/src/main/java/io/github/ust/mico/core/web/ServiceController.java

```diff
@@ -124,21 +124,34 @@
 
         if (!getDependers(service).isEmpty()) {
             throw new ResponseStatusException(HttpStatus.UNPROCESSABLE_ENTITY,
-                "Service '" + service.getShortName() + "' '" + service.getVersion() + "' has dependers, therefore it can't be deleted.");
+                    "Service '" + service.getShortName() + "' '" + service.getVersion() + "' has dependers, therefore it can't be deleted.");
         }
 
         serviceRepository.deleteServiceByShortNameAndVersion(shortName, version);
         return ResponseEntity.noContent().build();
     }
 
+    @DeleteMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}")
+    public ResponseEntity<Void> deleteServices(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName) {
```

Comment:
Maybe change function name to `deleteAllVersionsOfService` or something similar?